grunt/config/replace.js

```diff
@@ -37,6 +37,18 @@ module.exports = function (grunt, options) {
         if (configJson._xapi.hasOwnProperty('_activityID') && configJson._xapi._activityID == '') {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
+
+        // cmi5 is a profile of the xAPI specification and some systems, e.g. SCORM Cloud
+        // do not work well with both manifest types, so remove the one which is not being used.
+        if (configJson._xapi.hasOwnProperty('_specification')) {
```

Comment:
Hi @oliverfoster, it did feel like a bit of a hack!  I only put it here because there was already a mention to the `_xapi` extension on line 37.

Could we have Grunt enumerate a `/grunt` folder inside the plugins folder?